src/Http/Http.Extensions/src/RequestDelegateFactory.cs

```diff
@@ -669,7 +668,8 @@ private static Expression BindParameterFromValue(ParameterInfo parameter, Expres
                 }
 
                 // Allow nullable parameters that don't have a default value
-                var nullability = NullabilityContext.Create(parameter);
+                var nullabilityContext = new NullabilityInfoContext();
```

Comment:
Can this be created once per action and stored on the factory context so we don't need to make a new one per parameter?

Reply:
Yep!